src/jit/bitsetasuint64.h

```diff
@@ -139,6 +144,11 @@ class BitSetOps</*BitSetType*/ UINT64,
         return (bs1 & bs2) == 0;
     }
 
+    static void LivenessD(Env env, UINT64& in, const UINT64 def, const UINT64 use, const UINT64 out)
```

Comment:
Ah, I see--I thought that these were new functions and was only looking in the changes you made.